pkgs/by-name/op/openjpeg/package.nix

```diff
@@ -52,12 +63,17 @@ stdenv.mkDerivation rec {
     ++ lib.optionals jpipServerSupport [ curl fcgi ]
     ++ lib.optional (jpipLibSupport) jdk;
 
-  doCheck = (!stdenv.hostPlatform.isAarch64 && !stdenv.hostPlatform.isPower64); # tests fail on aarch64-linux and powerpc64
+  # tests fail on aarch64-linux and powerpc64
```

Comment:
I can check if these pass now on `aarch64-linux` if you’d like.

Reply:
thanks. I made a couple of tweaks to the `cmakeFlags` and removed `"-DCMAKE_INSTALL_NAME_DIR=\${CMAKE_INSTALL_PREFIX}/lib"` running the `passthru.tests` -- everything in `-dev` looks fine.